packages/sanity/src/core/form/inputs/PortableText/Compositor.tsx

```diff
@@ -294,7 +284,7 @@ export function Compositor(props: Omit<InputProps, 'schemaType' | 'arrayFunction
       <Editor
         hasFocus={hasFocus}
         hotkeys={editorHotkeys}
-        initialSelection={initialSelection}
+        initialSelection={null}
```

Comment:
does `null` have a specific meaning in this context or could the initialSelection prop be omitted here?

Reply:
It should be dropped! Tnx.